src/desktop/apps/auction_support/stylesheets/index.styl

```diff
@@ -42,19 +42,35 @@
       display flex
       justify-content center
       align-content center
+      // Vertically align the period.
+      line-height 1.5em
```

Comment:
Why do we need these separate `line-height`s for the text and the link?

Reply:
Gone now. I was struggling with line heights to make the link and period appear on the same line.